packages/shared-ui/scripts/generate-gallery-tokens.mjs

```
@@ -0,0 +1,357 @@
+
+// =============================================================================
+// CLI
+// =============================================================================
+
+/**
+ * Entry point: generate and write, or verify with `--check`.
+ * Exits non-zero in check mode when the committed file is stale.
+ */
+function main() {
+	const rendered = renderModule(buildUsageMap());
+	if (process.argv.includes('--check')) {
```

ESLint

[error] 'process' is not defined.

(no-undef)